manager/manager.go

```diff
@@ -687,6 +687,7 @@ func (self *manager) GetFsInfo(label string) ([]v2.FsInfo, error) {
 			Usage:      fs.Usage,
 			Available:  fs.Available,
 			Labels:     labels,
+			InodesFree: fs.InodesFree,
```

Comment:
Do you mean the FsInfo struct? I think we should try to maintain backwards compatibility between releases, but I think we don't need to worry about backwards compatibility with HEAD between releases.